shared/domains/chelonia/internals.js

```diff
@@ -338,6 +340,12 @@ export default (sbp('sbp/selectors/register', {
         const newEntry = await sbp('chelonia/private/queueEvent', contractID, () => {
```

Comment:
So, as a totally separate issue, I think we need to revisit this code:

```js
  'chelonia/queueInvocation': (contractID, sbpInvocation) => {
    // We maintain two queues, contractID, used for internal events (i.e.,
    // from chelonia) and public:contractID, used for operations that need to
    // be done when no internal operations are running (e.g., calls from a
    // contract that require to be done after a sync)
    // The following waits for the internal (contractID) queue to be finished
    // and then pushes the operation requested into the public queue
    // This ensures that all internal operations have finished before these
    // other selectors are called
    return sbp('chelonia/private/queueEvent', contractID, ['chelonia/private/noop']).then(() => sbp('chelonia/private/queueEvent', 'public:' + contractID, sbpInvocation))
  },
```

This comment doesn't make sense.

>     // We maintain two queues, contractID, used for internal events (i.e.,
>     // from chelonia) and public:contractID, used for operations that need to
>     // be done when no internal operations are running 

..

>     // This ensures that all internal operations have finished before these
>     // other selectors are called

These comments are wrong. In fact it allows the `sbpInvocation` to run at the same time as something is running on the internal queue.

I do not see an issue with directly pushing to the `contactID` queue. Now that we've stabilized the tests we should be able to test this theory.

Reply:
> I do not see an issue with directly pushing to the `contactID` queue. Now that we've stabilized the tests we should be able to test this theory.

After discussion, we reached the conclusion that although the comment needs to be fixed to be more accurate and accurately explain the reason for the different queue, we still might want to keep the different queue (because it might prevent developers from screwing up Chelonia's internal operaitons)